docs/source/basic_examples.rst

```diff
@@ -1022,6 +1022,38 @@ useful key for metal-containing molecules. You can use the
     stk.MoleculeMongoDb = _old_molecule_init
     pymongo.MongoClient = _mongo_client
 
+Creating New Topology Graphs with Existing Vertices
+===================================================
+
+The vertex classes that make up toplogy graphs in :mod:`.stk` can be
+accessed to speed up the implemention of new topology graphs.
+
+.. testcode:: creating-new-topology-graphs-with-existing-vertices
+
+    import stk
+
+    class NewCageTopology(stk.cage.Cage):
+
+        _vertex_prototypes = (
+            stk.cage.LinearVertex(0, [0, 0, 0]),
+            stk.cage.LinearVertex(1, [0, 1, 0]),
+        )
+
+        # Define Edges below.
+        _edge_prototypes = ()
```

Comment:
I think these examples should contain *some* edges, because that is an important part of creating new topology graphs /  cages. I think omitting them can be a little misleading.

Reply:
Done.